modules/webgl/src/context/context.js

```diff
@@ -166,15 +166,8 @@ export function destroyGLContext(gl) {
 export function resizeGLContext(gl, options = {}) {
   // Resize browser context
   if (gl.canvas) {
-    /* global window */
-    const devicePixelRatio = options.useDevicePixels ? window.devicePixelRatio || 1 : 1;
-
-    const width = `width` in options ? options.width : gl.canvas.clientWidth;
-    const height = `height` in options ? options.height : gl.canvas.clientHeight;
-
-    gl.canvas.width = width * devicePixelRatio;
-    gl.canvas.height = height * devicePixelRatio;
-
+    const devicePixelRatio = getDevicePixelRatio(options.useDevicePixels);
```

Comment:
Looks like a breaking change - `options.width` and `options.height` are no longer supported.

Reply:
`options.width` and `options.height` are not used by any one who is calling `resizeGLContext`, also these are not documented.
Are there any use cases I am missing? Do you think I should add this back?

Reply:
Let's be safe and add it back for now. You can make a comment to revisit the API in v8.

Reply:
Added.